modules/lang-painless/src/test/resources/rest-api-spec/test/painless/50_script_doc_values.yml

```diff
@@ -95,7 +95,7 @@ setup:
                     field:
                         script:
                             source: "doc.date.get(0)"
-    - match: { hits.hits.0.fields.field.0: '2017-01-01T12:11:12.000Z' }
+    - match: { hits.hits.0.fields.field.0: '2017-01-01T12:11:12Z' }
```

Comment:
Are we good with this change, as it changes the string representation of dates? I suppose this just calls `LocalTime.toString()` which only accounts for nanoseconds if they are greater than zero.

Reply:
Yes that is correct. We can mention it in the 7.0 docs when the bwc behavior is removed. I don't think it has any real impact, since with or without the sub second part, the adheres to ISO 8601.